src/utils/convertFromSecondToMinutes.js

```diff
@@ -0,0 +1,7 @@
+const convertFromSecondToMinutes = (time) => {
```

Comment:
personal idea: I think this name is needlessly long, we could go for `formatSecToMin` and it would transmit the same info